.github/workflows/master.yml

```diff
@@ -434,7 +434,6 @@ jobs:
         run: >-
           ./build/mvn ${MVN_OPT} clean install
           -Pflink-provided,hive-provided
-          -Pspark-3.2
```

Comment:
It's better to specify `spark-3.3`.

Apache Spark version of the kubernetes-it is currently definitive and unique, It should be consist with [kubernetes-it](https://github.com/apache/kyuubi/blob/master/integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/spark/SparkOnKubernetesTestsSuite.scala#L48) to to prevent serialization problems in k8s-client mode.

Before we finish this [TODO](https://github.com/apache/kyuubi/blob/master/integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/spark/SparkOnKubernetesTestsSuite.scala#L45), it's a good idea to specify a profile in case we change the default spark version of kyuubi and spark docker doesn't have mirroring issues with that version

Reply:
Yes, exactly.